screenpipe-server/src/bin/screenpipe-server.rs

```diff
@@ -25,6 +24,8 @@ use tracing_subscriber::{fmt, EnvFilter};
 use tracing::{info, debug, error};
 use tracing_appender::rolling::{RollingFileAppender, Rotation};
 use tracing_appender::non_blocking::WorkerGuard;
+use screenpipe_actions::run;
```

Comment:
you need to put i na beta block too

it shouldn't affect in any way the code when not using the beta flag